modules/schematics/collection.json

```diff
@@ -57,6 +57,13 @@
       "description": "Adds initial setup for state management"
     },
 
+    "selector": {
+      "aliases": ["s"],
```

Comment:
Let's change the alias to `se`, the `s` alias is already being used for services. https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/collection.json#L87

Reply:
Good catch, will do 👍